config/faf-clans/faf-clans.env

```diff
@@ -1,3 +1,7 @@
-WEBROOT=/usr/share/nginx/html
-TEMPLATE_NGINX_HTML=0
+API_BASE_URL=https://api.dev.faforever.com
+OAUTH_CLIENT_ID=83891c0c-feab-42e1-9ca7-515f94f808ef
+OAUTH_REDIRECT_URI=https://clans.faforever.com
```

Comment:
This is required by the clanapp and cannot be avoided. This only overrides the default from the clanapp and work also locally... https://github.com/FAForever/clans/blob/master/webpack.config.js#L10